db/models.py

```diff
@@ -0,0 +1,25 @@
+from sqlalchemy import Column, Integer, String, ForeignKey, Date
+from sqlalchemy.orm import relationship
+
+from db.engine import Base
+
+
+class Book(Base):
+    __tablename__ = "books"
+
+    id = Column(Integer, primary_key=True, index=True)
+    title = Column(String, index=True)
+    summary = Column(String)
+    publication_date = Column(Date)
+    author_id = Column(Integer, ForeignKey("authors.id"))
+
+    author = relationship("Author", back_populates="books")
+
+
+class Author(Base):
+    __tablename__ = "authors"
+
+    id = Column(Integer, primary_key=True, index=True)
+    name = Column(String, unique=True, index=True)
```

Comment:
I see you didn't push a new migration file because in this migration summary is still nullable, but in your code no ;)